hcatalog/server-extensions/src/main/java/org/apache/hive/hcatalog/messaging/AlterTableMessage.java

```diff
@@ -42,4 +42,6 @@ public HCatEventMessage checkValid() {
   }
 
   public abstract Long getWriteId();
+
+  public abstract String getLocOwner();
```

Comment:
Shall use a complete name getDataLocationOwner. It is not too long. Same for other places.